engine/src/graphics/renderer.cpp

```diff
@@ -7,11 +7,12 @@
 namespace gen
 {
 
-	Renderer::Renderer(const Window & window, const char * const appName, const u32 appVersion)
+	Renderer::Renderer(const char * const appName, const u32 appVersion)
+		: m_device(std::make_unique<Device>(appName, appVersion, "Genesis Engine", VK_API_VERSION_1_2)),
+		  m_swapchain(std::make_unique<Swapchain>(Window::getInstance(), *m_device, m_device->getSurface())),
```

Comment:
Shouldn't this be `VK_API_VERSION_1_3`?

Reply:
Oops accidentally typed 2 instead of 3 xD